src/webextension/list/components/item-fields.css

```diff
@@ -32,3 +32,11 @@
 .notes {
   min-height: 60px;
 }
+
+.notes-read-only {
+  min-height: 73px;
+}
+
+.password {
+  font-weight: normal;
```

Comment:
I *think* this is unnecessary now (at least on Linux); if it's actually required to make things look ok on Windows though, then I'm fine with keeping it.